test/cmd/newapp.sh

```diff
@@ -69,7 +69,9 @@ os::cmd::expect_success_and_text 'oc new-app ruby-helloworld-sample --param MYSQ
 # verify we can create from a template when some objects in the template declare an app label
 # the app label should still be applied to the other objects in the template.
 os::cmd::expect_success_and_text 'oc new-app -f test/fixtures/template-with-app-label.json -o yaml' 'app: ruby-sample-build'
-os::cmd::expect_success_and_text 'oc new-app -f test/fixtures/template-with-app-label.json -o yaml' 'app: myapp'
+# TODO: This is currently broken because of the changes in pkg/template that
```

Comment:
in it squashed in the interesting changes.... we changed the label manipulation to always override the labels if they exists (there is not way to do "merge" sanely with the latest kube.